azure-pipelines.yml

```diff
@@ -347,17 +347,23 @@ jobs:
     inputs:
       packageFeedSelector: 'nugetOrg'
 
-  - task: DownloadSecureFile@1
-    name: PsExec
-    displayName: 'Download PsExec.exe'
+  - task: PowerShell@2
+    displayName: Install WinGet and PsExec
     inputs:
-      secureFile: 'PsExec.exe'
+      pwsh: true
+      targetType: 'inline'
+      script: |
+        Install-Module Microsoft.WinGet.Client -Repository PSGallery -Force
+        Repair-WingetPackageManager
+        winget install --id Microsoft.Sysinternals.PsTools --accept-source-agreements

```

Comment:
Nit, not needed: Could also specify `-s winget` instead of accepting the agreements for msstore